charts/datadog/templates/rbac.yaml

```diff
@@ -2,13 +2,14 @@
 apiVersion: {{ template "rbac.apiVersion" . }}
 kind: ClusterRole
 metadata:
+  name: {{ template "datadog.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
I think ClusterRole doesn't need a namespace

Reply:
:100: fixed in 959a9f5